### ISSUE TITLE
File Browser Icon is not showing  / Heading Over write

### ISSUE THREAD

**Issue by Ajithpalotte** [coder/modules#372]
Hello Team,

I Have installed the file browser in my workspace using the module, but i am not getting the proper logo and name for the File browser(please refer the below screenshot).  Can you please help me to fix the issue

![Image](https://github.com/user-attachments/assets/b28f0b22-db7a-4718-a70c-990be3f5a859)

Code is used to install file browser

```
locals
{
filebrowser_port = "13339"
}

module "filebrowser" {
  count          = data.coder_workspace.me.start_count
  source   = "registry.coder.com/modules/filebrowser/coder"
  version  = "1.0.23"
  agent_id = coder_agent.dev[0].id
  port = local.filebrowser_port
  slug         = "filebrowser"
  folder   = "/home/coder/"
}

```

**Comment by ethanndickson** [comment]
What version of Coder are you running? This should be fixed in 2.17 and above via https://github.com/coder/coder/commit/5a6d23a4a3a2225e3440f4a2dd83b3fc1fdffaa4

**Comment by Ajithpalotte** [comment]
Hello @matifali,

I am using the  filebrowser module with version   version  = "1.0.25" after updating the version 2.17  i am getting the below error.




```
  | Create one by running  terraform init  in your template directory.
=== ✔ Queued [81ms]
==> ⧗ Running
==> ⧗ Running
=== ✔ Running [18ms]
==> ⧗ Setting up
=== ✔ Setting up [0ms]
==> ⧗ Parsing template parameters
=== ✔ Parsing template parameters [25ms]
==> ⧗ Detecting persistent resources
2024-12-19 14:18:47.657+05:30 Error: Unresolvable module version constraint
2024-12-19 14:18:47.657+05:30   on main.tf line 195:
2024-12-19 14:18:47.657+05:30  195: module "filebrowser" {
2024-12-19 14:18:47.657+05:30 There is no available version of module
2024-12-19 14:18:47.657+05:30 "registry.coder.com/modules/filebrowser/coder" (main.tf:195) which matches
2024-12-19 14:18:47.657+05:30 **the given version constraint. The newest available version is 1.0.25.**
=== ✔ Detecting persistent resources [834ms]
==> ⧗ Cleaning Up
=== ✘ Cleaning Up [21ms]
=== ✘ Cleaning Up [212ms]
```

I am using the below code

```
module "filebrowser" {
  count    = data.coder_workspace.me.start_count
  source   = "registry.coder.com/modules/filebrowser/coder"
  version  = "1.0.25"
  agent_id = coder_agent.dev[0].id
  port     = local.filebrowser_port
  slug     = "filebrowser"
  folder   = "/home/coder/"
  share    = "owner"

}
```
